src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const copy = {};
```

Comment:
It is technically a copy, but I'd suggest using a more descriptive name for a variable 
(Checklist 3)